### PR TITLE
feat: add default field visualization

### PR DESCRIPTION
**Pull request by DittrichLucas**
**- What I did**

Show the default input value in parentheses at the end of the question.

default

This was necessary to make it more evident to the user that, by not informing value in an input that has a default value, it will be used.

Initially, the types of input that accept this improvement are: text and int. For the other types (bool, email, list, multiline and password) this improvement was not efficient.

**- How to verify it**

Run a formula that has input with default value.

**- Description for the changelog**

Show the default input value in parentheses at the end of the question.

### CODE REVIEW

**Review by henriquemoraeszup — COMMENTED** [review]
I am a bit confused, I thought we just needed to display the text of the default value in case the user defined a default value. Maybe inputs like list or bool do not even require this refactor. Imagine you will always display a list of items, if the user presses enter straight away, then the first item is always selected, much like a default value.

What do you think?

File: pkg/cmd/tutorial.go [hunk 1]
```diff
@@ -87,7 +87,7 @@ func (o tutorialCmd) runPrompt() CommandRunnerFunc {
 		tutorialStatusCurrent := tutorialHolder.Current
 		fmt.Println("Current tutorial status: ", tutorialStatusCurrent)
 
-		response, err := o.List(msg, statusTypes)
+		response, err := o.List(msg, statusTypes, statusTypes[0])
```
Comment (henriquemoraeszup) [inline pkg/cmd/tutorial.go:90]: Is this list staying?

File: pkg/cmd/upgrade.go [hunk 2]
```diff
@@ -67,7 +67,7 @@ func (u UpgradeCmd) runFunc() CommandRunnerFunc {
 		if !u.file.Exists(metric.FilePath) {
 
 			options := []string{AcceptMetrics, DoNotAcceptMetrics}
-			choose, err := u.input.List(AddMetricsQuestion, options)
+			choose, err := u.input.List(AddMetricsQuestion, options, options[1])
```
Comment (henriquemoraeszup) [inline pkg/cmd/upgrade.go:70]: This list as well?

File: pkg/cmd/create_formula.go [hunk 3]
```diff
@@ -103,7 +103,7 @@ func (c createFormulaCmd) runPrompt() CommandRunnerFunc {
 			return err
 		}
 
-		lang, err := c.inList.List("Choose the language: ", languages)
+		lang, err := c.inList.List("Choose the language: ", languages, "")
```
Comment (henriquemoraeszup) [inline pkg/cmd/create_formula.go:106]: Are we removing default values from lists?

File: pkg/cmd/init.go [hunk 4]
```diff
@@ -260,7 +260,7 @@ You can view our Privacy Policy (http://insights.zup.com.br/politica-privacidade
 }
 
 func (in initCmd) setRunnerType() error {
-	selected, err := in.List(SelectFormulaTypeQuestion, formula.RunnerTypes)
+	selected, err := in.List(SelectFormulaTypeQuestion, formula.RunnerTypes, formula.RunnerTypes[0])
```
Comment (henriquemoraeszup) [inline pkg/cmd/init.go:263]: How about this list?

**Review by henriquemoraeszup — COMMENTED** [review]
I see we have some lists still, is this intentional? If we remove them we actually reduce the number of files edited, which is good.

Another thing, if we had a new component like `InputTextDefault` we could avoid refactoring the code in multiple files by changing only the formula one, do you think its valid? I do not have a strong opinion, just to think about it =]

File: pkg/prompt/int.go [hunk 5]
```diff
@@ -31,20 +31,20 @@ func NewSurveyInt() SurveyInt {
 	return SurveyInt{}
 }
 
-func (SurveyInt) Int(name string, helper ...string) (int64, error) {
+func (SurveyInt) Int(name string, defaultValue string, helper ...string) (int64, error) {
```
Comment (victorschumacherzup) [inline pkg/prompt/int.go:34]: is possible to declare the parameter only once here? like: 
``` go 
func Int(name, defaultValue string, helper ...string)
```

File: pkg/prompt/int.go [hunk 6]
```diff
@@ -31,20 +31,20 @@ func NewSurveyInt() SurveyInt {
...
 	var value string
+	var input = &survey.Input{Message: name, Default: defaultValue}
```
Comment (victorschumacherzup) [inline pkg/prompt/int.go:36]: i think a simplified variable declaration would be cleaner ```:=```

File: pkg/prompt/text.go [hunk 7]
```diff
@@ -26,13 +26,15 @@ func NewSurveyText() SurveyText {
 	return SurveyText{}
 }
 
-func (SurveyText) Text(name string, required bool, helper ...string) (string, error) {
+func (SurveyText) Text(name string, required bool, defaultValue string, helper ...string) (string, error) {
 
 	var value string
+	var input = &survey.Input{Message: name, Default: defaultValue}
```
Comment (victorschumacherzup) [inline pkg/prompt/text.go:32]: same observations here

File: pkg/cmd/create_formula.go [hunk 8]
```diff
@@ -286,12 +286,12 @@ func FormulaWorkspaceInput(
...
 	var wspace formula.Workspace
 	if selected == newWorkspace {
-		workspaceName, err = inText.Text("Workspace name: ", true)
+		workspaceName, err = inText.Text("Workspace name: ", true, "")
 		if err != nil {
 			return formula.Workspace{}, err
 		}
 
-		workspacePath, err = inText.Text("Workspace path (e.g.: /home/user/github):", true)
+		workspacePath, err = inText.Text("Workspace path (e.g.: /home/user/github):", true, "")
```
Comment (victorschumacherzup) [inline pkg/cmd/create_formula.go:294]: The third arg ir necessary? I think you can pass only two because ```Text``` receive a ```...string```

**Comment by lucasdittrichzup** [comment]
due to the amount of difference to be passed and the scope changes, I am closing this PR to open a new

**Pull request closed without merging**